src/commands/hardis/lint/unused.ts

```diff
@@ -0,0 +1,188 @@
+/* jscpd:ignore-start */
+import { flags, SfdxCommand } from "@salesforce/command";
+import { Messages } from "@salesforce/core";
+import { AnyJson } from "@salesforce/ts-types";
+import * as glob from "glob-promise";
+import { uxLog } from "../../../common/utils";
+import * as fs from "fs-extra";
+//TODO getConfig Unused how to use this ?
+// import { getConfig } from "../../../config";
+import * as xml2js from "xml2js";
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = Messages.loadMessages("sfdx-hardis", "org");
+
+export default class Unused extends SfdxCommand {
+  public static title = "check permission access";
+  public static description = "Check if elements(apex class and field) are at least in one permission set";
+  public static examples = [
+    "$ sfdx hardis:lint:access",
+    '$ sfdx hardis:lint:access -e "ApexClass:ClassA, CustomField:Account.CustomField"',
+    '$ sfdx hardis:lint:access -i "PermissionSet:permissionSetA, Profile"',
+  ];
+  /* jscpd:ignore-start */
+  protected static flagsConfig = {
+    debug: flags.boolean({
+      char: "d",
+      default: false,
+      description: messages.getMessage("debugMode"),
+    }),
+    websocket: flags.string({
+      description: messages.getMessage("websocket"),
+    }),
+    skipauth: flags.boolean({
+      description: "Skip authentication check when a default username is required",
+    }),
+  };
+  /* jscpd:ignore-end */
+
+  // Comment this out if your command does not require an org username
+  protected static requiresUsername = false;
+  // Comment this out if your command does not support a hub org username
+  protected static supportsDevhubUsername = false;
+  // Set this to true if your command requires a project workspace; 'requiresProject' is false by default
+  protected static requiresProject = true;
+
+  private parser: xml2js.Parser;
+  private directories = [
+    "force-app/main/default/lwc",
+    "force-app/main/default/flows",
+    "force-app/main/default/classes",
+    "force-app/main/default/email",
+    "force-app/main/default/aura",
+    "force-app/main/default/flexipages",
+    "force-app/main/default/quickActions",
+    "force-app/main/default/objects",
+    "force-app/main/default/pages",
+    "force-app/main/default/staticresources",
+  ];
+  private objectFieldsDirectory = "force-app/main/default/objects/**/fields/*.*";
```

Comment:
You can find the directori(e)s by searching for .field-meta.xml files :)
Same remark for PS & labels ^^